cmd/sansshell-server/server/server.go

```diff
@@ -73,6 +76,14 @@ func WithPolicy(policy string) Option {
 	})
 }
 
+// WithTlsConfig applies a supplied tls.Config object to the gRPC server.
```

Comment:
This isn't the only place. Also need to do cmd/proxy-server/server/server.go since it does a similar thing (this wrapper is only for the sansshell-server part)

Reply:
Thanks for pointing that out. I think that I have gotten the other places as well.